src/validation.cpp

```diff
@@ -788,13 +788,15 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
         size_t nConflictingSize = 0;
         uint64_t nConflictingCount = 0;
         CTxMemPool::setEntries allConflicting;
+        bool fReplacementTransaction = false;
```

Comment:
nit: you may as well make this `const bool fReplacementTransaction = setConflicts.size()` under the lock & make the if below check fReplacementTransaction.